linchpin/defaults/schemas/schema_v4.json

```diff
@@ -108,8 +108,35 @@
                                  "keyvalue": {
                                      "type": "array"
                                  },
-                                 "hostrequire": {
-                                     "type": "array"
+                                 "hostrequires": {
+                                     "type": "array",
+                                     "items": {
+                                         "not": {
+                                             "allOf": [
+                                                 {"required": ["force", "tag"]}
+                                             ]
+                                         },
```

Comment:
> Is there any specific reason you used allof line 115 of schemav4 instead of oneof ?

It's a under a "not", and I'm trying to express than you can have anyOf objects with a "tag" key or objects with a "force" key, and also that you can not have allOf objects with a "tag" key and objects with a "force" key. It only makes sense to use "force" once, but it's acceptable to have multiple tags.

oneOf might do what I want here, but I think I'm going to break this out into its own issue, and pull the mutual exclusion stuff out of this PR. The included docs for beaker topologies, at least, explain that any included "tag" objects will be ignored if "force" is used.